src/DependencyInjection/TransportProvider.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Symfony\Messenger\DependencyInjection;
+
+use Aws\EventBridge\EventBridgeClient;
+use Aws\Sns\SnsClient;
+use Aws\Sqs\SqsClient;
+use Bref\Symfony\Messenger\Service\EventBridge\EventBridgeTransportFactory;
+use Bref\Symfony\Messenger\Service\Sns\SnsTransportFactory;
+use Bref\Symfony\Messenger\Service\Sqs\SqsTransportFactory;
+
+class TransportProvider
+{
+    public static function getAllServices(): array
+    {
+        return [
+            'sns' => [
+                'default_client' => SnsClient::class,
+                'package' => 'aws/aws-sdk-php',
+                'transport_factory' => SnsTransportFactory::class,
+            ],
+            'sqs' => [
+                'default_client' => SqsClient::class,
+                'package' => 'aws/aws-sdk-php',
```

Comment:
Oh, this class looks a bit strange... It is like it is preparing to allow other packages than `aws/aws-sdk-php` to to be used with this bundle. 
=)